python/versions.bzl

```
@@ -798,29 +802,33 @@ TOOL_VERSIONS = {
             "s390x-unknown-linux-gnu-freethreaded": "python/install",
             "x86_64-apple-darwin-freethreaded": "python/install",
             "x86_64-pc-windows-msvc-freethreaded": "python/install",
+            "aarch64-pc-windows-msvc": "python/install",
```

@AustinSchuh, Is this correct? I think this may need to be

```suggestion
            "aarch64-pc-windows-msvc": "python",
```